autonomic-administration-algorithms/src/main/java/br/com/autonomiccs/autonomic/administration/algorithms/impl/ConsolidationScoredPreferenceForSmallHosts.java

```diff
@@ -40,6 +40,11 @@
  */
 public class ConsolidationScoredPreferenceForSmallHosts extends ConsolidationAlgorithmBase {
 
+    /**
+     * This constant is the same as {@link ConsolidationAlgorithmBase#BYTES_TO_MEGA_BYTES}, but it is of type double.
+     */
+    protected final static double BYTES_TO_MEGA_BYTES_AS_DOUBLE = ConsolidationAlgorithmBase.BYTES_TO_MEGA_BYTES * 1d;
+
```

Comment:
I didn't understand why to do `BYTES_TO_MEGA_BYTES_AS_DOUBLE = ConsolidationAlgorithmBase.BYTES_TO_MEGA_BYTES * 1d` (instead of `BYTES_TO_MEGA_BYTES_AS_DOUBLE = ConsolidationAlgorithmBase.BYTES_TO_MEGA_BYTES`), is there an issue pointed by _Coverity_? Thanks.